### PR TITLE
Missing code of conduct

### PR DESCRIPTION
**Pull request by sidhyatikku**
Added missing Code of Conduct #143 

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [ X I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Comment by sidhyatikku** [comment]
@TheSisb requesting review regarding #143

File: README.md [hunk 1]
```diff
@@ -15,7 +15,7 @@ Build things 🎉
 
 ## Contributing
 
-[Please check the contribution docs](https://github.com/twilio-labs/paste/blob/master/CONTRIBUTING.md)
+Before contributing, please make sure that you read our [Contributing Guidelines](/CONTRIBUTING.md) and agree with our [Code of Conduct.](/CODE_OF_CONDUCT.md)
```
Comment (TheSisb) [inline README.md]: I don't think we should remove the link to our contribution guide.  Just add this underneath.  

```
## Code of Conduct

Please be aware that this project has a [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md). The tl;dr is to just be excellent to each other ❤️
```

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
+# Contributor Covenant Code of Conduct
```
Comment (TheSisb) [inline CODE_OF_CONDUCT.md]: Let's not copy this over to our repo.  I like linking to the Twilio Labs one because if it ever changes, this won't go stale.

**Comment by TheSisb** [comment]
Thanks!

**Pull request merged**